docs/timezone.md

```diff
@@ -45,6 +47,51 @@ tz1.getTransitions(now).next().done;  // => true
 tz2.getTransitions(now).next().done;  // => false
 ```
 
+## Temporal.TimeZone.from(thing: string | object) : Temporal.TimeZone
+
+**Parameters:**
+- `thing` (string or object): A `Temporal.TimeZone` object or a string from which to create a `Temporal.TimeZone`.
+
+**Returns:** a new `Temporal.TimeZone` object.
+
+This static method creates a new time zone from another value.
+If the value is a string, it can be:
+- a string that is accepted by `new Temporal.TimeZone()`;
+- a string in the ISO 8601 format including a time zone part, optionally extended with an IANA time zone name in square brackets appended to it;
```

Comment:
```suggestion
- a string in the ISO 8601 format including a UTC offset, optionally extended with an IANA time zone name in square brackets appended to it;
```

Reply:
Not sure I agree with this change. A time zone part can be a UTC offset optionally extended with an IANA name, but it can also be `Z`.

Reply:
Surely "2020-01-25T15:15Z[Etc/UTC]" is invalid, right? "-05:00" is not a time zone, it is a difference between local time and UTC (which can be followed by a bracketed time zone name). And "Z" in that position is also not a time zone, it is a UTC designator (which cannot be).

Reply:
How about "a string in the ISO 8601 format including a time zone part" and then clarify in the text below that a time zone part can be "either the UTC designator `Z`, or a UTC offset optionally extended with an IANA name in brackets"?

Reply:
That structure definitely works, but I think I would still call it an "offset part" rather than a "time zone part", because it's fundamentally coming from ISO 8601 as a UTC offset and only being _extended_ to include time zone information.